ui/appui-react/src/appui-react/content/ContentLayout.scss

```diff
@@ -27,26 +27,71 @@ $resizer-size-large: 14px;
     margin: 0;
     transition: all var(--iui-duration-1) ease;
 
-    &:hover {
-      background-color: var(--iui-color-border-accent);
-    }
-
     &.horizontal {
-      height: $resizer-size;
-      min-height: $resizer-size;
+      height: 1px;
```

Comment:
`$resizer-size` is no longer used? Either remove it or just change the value to 1px and continue using it.